.travis.yml

```
@@ -18,8 +18,8 @@ sudo: required
 dist: trusty
 
 php:
-  - 5.6
   - 7.2
```

```suggestion
   - 7.1
```

This would allow us to cover the whole compatibility range.